src/components/Contacts/AddContact.js

```diff
@@ -147,17 +155,15 @@ const AddContact = ({ classes, addNewContact }) => {
               <FormHelperText className={classes.formHelperText}>
                 {errors.email_error || null}
               </FormHelperText>
-
-              <TextField
-                required
-                id="phone"
-                label="Primary Phone"
+              <MuiPhoneNumber
                 name="phone_primary"
+                label="Primary Phone"
+                defaultCountry={'us'}
                 value={values.phone_primary}
-                onChange={handleChange}
-                className={classes.textField}
+                onChange={handlePhoneInput}
                 InputLabelProps={inputLabelProps}
                 InputProps={inputProps}
+                className={classes.textField}
```

Comment:
In playing with the component, I noticed that it displays every area code as an option in the dropdown, which is kind of weird
<img width="487" alt="Screen Shot 2019-10-17 at 10 22 57 AM" src="https://user-images.githubusercontent.com/618075/67024249-b5298280-f0c9-11e9-8672-a8dad44e4151.png">
It looks like we can disable this behavior by adding the `disableAreaCodes` prop, which I think we should probably do

Reply:
Done